libs/checkpoint/langgraph/store/base/__init__.py

```diff
@@ -463,6 +463,13 @@ class PutOp(NamedTuple):
         ]
         ```
     """
+    ttl: Optional[float] = None
```

Comment:
Could expand to a dict {"refresh_on": "rw", "after_minutes": foo} if we want it to be configurable.